py/src/pathfinder_worker/call.py

```diff
@@ -384,8 +384,8 @@ def do_loop(connection: sqlite3.Connection, input_gen, output_file):
 
                 if exc.message:
                     message = exc.message
-                    if len(message) > 200:
-                        message = message[:197] + "..."
+                    # if len(message) > 200:
+                    #     message = message[:197] + "..."
```

Comment:
Probably left in by mistake?

Reply:
Good catch, that was for "bebugging" only :+1: